app/jobs/ensure_scoped_daily_summaries_job.rb

```diff
@@ -1,5 +1,5 @@
 class EnsureScopedDailySummariesJob < ApplicationJob
-  queue_as :tail
+  queue_as :ensure_scoped_daily_summaries
```

Comment:
Are we still using the tail queue? If not we should remove it from our Sidekiq config